### DISCUSSION TITLE
Thanks for bringing the extension back to life!

### DISCUSSION THREAD

**Discussion by waldyrious** [WengerK/vscode-highlight-bad-chars#25] [Kind words]
Hi @ItalyPaleAle. I didn't notice any public conversations regarding how you became a maintainer for this project, but I assume @WengerK offered to make you one after you submitted #18 (Wow, that was just 3 days ago? With so much activity going on, it felt like a lot more time had passed!). The extension had been dormant for a few years (last commit was in June 2018) but after you got access, you've been thoroughly trimming the backlog of issues and PRs, and implementing long-standing community requests.

I just wanted to say thank you for your excellent work as a maintainer of this project! People like you are the reason I keep following projects long after they're dormant, or keep PRs open for months, sometimes years. That's a great demonstration of the spirit of open source! Keep up the amazing work, major props for stepping up! And props to @WengerK as well, for passing the torch! You guys rock! :clap: :heart: :rocket:

**Comment by WengerK** [comment]
All credits goes to @ItalyPaleAle. Thank for helping and tackling all those long-term issues.

We will attempt to release a new version tomorrow morning

**Comment by WengerK** [comment]
`0.0.4` has been release right now !
